libs/commons-database/src/main/kotlin/gov/cdc/ocio/database/health/HealthCheckDynamoDb.kt

```diff
@@ -11,27 +10,22 @@ import software.amazon.awssdk.services.dynamodb.DynamoDbClient
  */
 class HealthCheckDynamoDb: HealthCheckSystem("Dynamo DB") {
 
+    // Lazily initialized DynamoDB client
+    private val dynamoDbClient: DynamoDbClient by lazy {
+        DynamoDbClient.builder().build()
+    }
     /**
      * Perform the dynamodb health check operations.
      */
     override fun doHealthCheck() {
         val result = runCatching {
-            getDynamoDbClient().listTables()
+            dynamoDbClient.listTables()
```

Comment:
Can this throw an exception if the client is unavailable?  If so, try catch should be added.